### DISCUSSION TITLE
Error accessing the Hyperion management panel.

### DISCUSSION THREAD

**Discussion by rintenten** [hyperion-project/hyperion.ng#1736] [Q&A]
Hello, I'm configuring the management panel of Hyperion installed on my Raspberry Pi Zero card. After adjusting the settings as needed, I click on "Restart" in the Hyperion settings to activate them. However, afterward, I can't access 192.168.1.x:8090 again. It times out (1/100, 20/100...), and I can only access Hyperion again by restarting the Raspberry Pi. I can't figure out the reason behind this issue. I would greatly appreciate any suggestions you may have.

**Comment by Lord-Grey** [comment]
To better understand the scenario where the issue happened...

Could you share what OS you are using and how you installed what version of Hyperion?
What settings did you update specifically before restarting.

Note: There is no need to restart Hyperion after updating settings, as updates are applied immediately after saving them.

**Comment by rintenten** [comment]
Thank you for replying to my message. I tried many ways and finally solved the problem. However, I have tried so many things that I don't know what's causing it. When I deactivate and reactivate the active USB capture button, the LEDs turn on. There was no need to reboot.
